bin/run.sh

```diff
@@ -1,13 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 
 # Synopsis:
 # Automatically tests exercism's JS track solutions against corresponding test files.
 # Takes two arguments and makes sure all the tests are run
 
 # Arguments:
 # $1: exercise slug
-# $2: path to solution folder (with trailing slash)
-# $3: path to output directory (with trailing slash)
+# $2: path to solution folder (without trailing slash)
+# $3: path to output directory (without trailing slash)
```

Comment:
This is wrong. it should be "with". See the spec: https://github.com/exercism/automated-tests/blob/master/docs/interface.md

Reply:
@iHiD well, that's not what was happening when @joshgoebel was testing this out. Everthing came in _without_ a trailing slash, breaking all the scripts.

Reply:
OK. So that's a bug in the development code that's shelling out to Docker then. But the spec of the actual thing in production hasn't changed. I'll fix the invoker.

Reply:
Fixed here: https://github.com/exercism/local-tooling-webserver/commit/45605e057ac7d3b1ac75036cc93a9adf39f483e7#diff-65b7416d6219182ef0492412293818f5R18-R19

```suggestion
# $2: path to solution folder (with trailing slash)
# $3: path to output directory (with trailing slash)
```

Reply:
Ok, if you all say so! :-)